includes/admin/views/html-admin-page-addons.php

```diff
@@ -25,19 +25,41 @@
 
 	<?php if ( $sections ) : ?>
 		<ul class="subsubsub">
-			<?php foreach ( $sections as $section_id => $section ) : ?>
-				<li><a class="<?php echo $current_section === $section_id ? 'current' : ''; ?>" href="<?php echo admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section_id ) ); ?>"><?php echo esc_html( $section->title ); ?></a><?php echo ( end( $section_keys ) !== $section_id ) ? ' |' : ''; ?></li>
+			<?php foreach ( $sections as $section ) : ?>
+				<li>
+					<a
+						class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
+						href="<?php echo admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ); ?>">
+						<?php echo esc_html( $section->label ); ?>
+					</a>
+				</li>
 			<?php endforeach; ?>
 		</ul>
-		<br class="clear" />
-		<?php if ( 'featured' === $current_section ) : ?>
+
+		<?php if ( isset( $_GET['search'] ) ) : ?>
+			<h1 class="search-form-title" >
+				Showing Search results for:
+				<strong><?php echo esc_html( $_GET['search'] ); ?></strong>
+			</h1>
+		<?php endif; ?>
+
+		<form class="search-form" method="GET">
+			<button type="submit">
+				<span class="dashicons dashicons-search"></span>
+			</button>
+			<input type="text" name="search" value="" placeholder="Enter a search term and press enter">
```

Comment:
Need to internationalize this placeholder. Perhaps also set the value to the current search term if not empty?